calico_containers/tests/st/docker_host.py

```diff
@@ -79,3 +80,6 @@ def assert_powerstrip_up(self):
         """
         powerstrip = partial(self.execute, "docker ps", use_powerstrip=True)
         retry_until_success(powerstrip, ex_class=ErrorReturnCode)
+
+    def create_workload(*args, **kwargs):
```

Comment:
Writing a function with arguments like this is basically saying "I'm not telling you anything about how to use this function!"

I get that in this specific case, the constructor for Workload essentially defines the arguments, but I doubt python IDEs are smart enough to figure that out.  I for one use auto-completion all the time.